nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -1068,16 +1097,13 @@ public ImmutableList<StreamTypeRecord> customStreamNullabilitySpecs() {
     }
   }
 
-  /**
-   * A view of library models optimized to make lookup of {@link
-   * com.sun.tools.javac.code.Symbol.MethodSymbol}s fast
-   */
+  /** A view of library models optimized to make lookup of {@link Symbol.MethodSymbol}s fast */
   private static class OptimizedLibraryModels {
 
     /**
      * Mapping from {@link MethodRef} to some state, where lookups first check for a matching method
      * name as an optimization. The {@link Name} data structure is used to avoid unnecessary String
-     * conversions when looking up {@link com.sun.tools.javac.code.Symbol.MethodSymbol}s.
+     * conversions when looking up {@link Symbol.MethodSymbol}s.
```

Comment:
Why this change?

Reply:
I am not sure why this went in there, It was probably the IDE that did something, I'll revert it.